# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/frontend/BytecodeCompiler.cpp
# Commit: adb4d60ce68c
# Full Hash: adb4d60ce68c8459927947f2703adf99e49e8ec0
# Author: Tooru Fujisawa <arai_a@mac.com>
# Date: 2024-12-05 09:58:04
# Regressor Bug: 1933681
# File Overlap Count: 1
# Description:
#   Bug 1933681 - Part 6: Remove unused profiler integration. r=nbp
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D231021
# ==============================================================================

diff -r 2ac97be10da5 -r adb4d60ce68c js/src/frontend/BytecodeCompiler.cpp
--- a/js/src/frontend/BytecodeCompiler.cpp	Wed Dec 04 23:44:34 2024 +0000
+++ b/js/src/frontend/BytecodeCompiler.cpp	Wed Dec 04 23:44:35 2024 +0000
@@ -1508,12 +1508,6 @@
   }
 
   if (output.is<RefPtr<CompilationStencil>>()) {
-    Maybe<AutoGeckoProfilerEntry> pseudoFrame;
-    if (maybeCx) {
-      pseudoFrame.emplace(maybeCx, "script emit",
-                          JS::ProfilingCategoryPair::JS_Parsing);
-    }
-
     auto extensibleStencil =
         fc->getAllocator()->make_unique<frontend::ExtensibleCompilationStencil>(
             std::move(compilationState));
